plugins/module_utils/pacemaker.py

```diff
@@ -59,6 +64,7 @@ def pacemaker_runner(module, cli_action=None, **kwargs):
             resource_operation=cmd_runner_fmt.as_func(fmt_resource_operation),
             resource_meta=cmd_runner_fmt.stack(cmd_runner_fmt.as_opt_val)("meta"),
```

Comment:
Unrelated to this change, but I would suggest you add tests for this option - I suspect this is not working as expected.